### PR TITLE
PMM-6212 Add goreleaser installation to docker image

### PR DESCRIPTION
**Pull request by Sudokamikaze**
Rebased from original PR

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1080-5185018
client docker - perconalab/pmm-client-fb:PR-1080-5185018
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1080-5185018.tar.gz

**Review by EvgeniyPatlan — DISMISSED** [review]

**Review by AlekSi — DISMISSED** [review]
:shipit:

File: build/rpmbuild-docker/Dockerfile [hunk 1]
```diff
@@ -23,16 +23,23 @@ RUN yum clean all && rm -rf /var/cache/yum
 
 ENV GO_VERSION 1.14.4
```
Comment (AlekSi) [inline build/rpmbuild-docker/Dockerfile:24]: https://github.com/Percona-Lab/pmm-submodules/pull/1079

Comment (Sudokamikaze) [reply build/rpmbuild-docker/Dockerfile:24]: Sure, it'll be rebased

Comment (Sudokamikaze) [reply build/rpmbuild-docker/Dockerfile]: @AlekSi @EvgeniyPatlan Rebased in 23e70a5

**Review by EvgeniyPatlan — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1080-23e70a5
client docker - perconalab/pmm-client-fb:PR-1080-23e70a5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1080-23e70a5.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1080-ed0d1a4
client docker - perconalab/pmm-client-fb:PR-1080-ed0d1a4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1080-ed0d1a4.tar.gz

**Pull request merged**